File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    // prompt user if hight is less than one or greater than eight
    while (n < 1 || n > 8);
    // # per row
    for (int i = 1; i <= n; i++)
    {
        // number of spaces on the left
        for (int s = 1; s < n - i + 1; s++)
        {

            printf(" ");
        }

     // left column
        for (int j = 1; j <= i; j++)
        {


            printf("#");
        }
        // print two spaces between the columns
        printf("  ");
        // right column
        for (int j = 1; j <= i; j++)
        {

            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

height = 0
# prompting user for height if input is not an int between 1 and 8
while height < 1 or height > 8:
    height = get_int("Height:")

for i in range(height):
    for j in range(height-i-1):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")
    for s in range(2):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")
    print()